common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -339,6 +348,12 @@ def test_member_of_a_team(self):
             self.my_teams_page.visit()
         self.verify_teams(self.my_teams_page, teams)
 
+    def test_multiple_team_members(self):
```

Comment:
If you have time, can you add a flaky decorator to this and run the new test 10 times? This is something the team wants to do with all new tests to help avoid introducing new flaky tests.